addons/l10n_zm/data/template/account.tax-zm.csv

```diff
@@ -0,0 +1,25 @@
+"id","description","invoice_label","type_tax_use","name","amount_type","amount","tax_group_id","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id"
+"zm_tax_sale_export_0","Export Rate","0%","sale","0% EX","percent","0.0","zm_tax_group_0","base","invoice",
```

Comment:
no tax report for Zambia?

Reply:
Couldn't find any official tax report no.

Reply:
I actually was able to find a somewhat official tax report when doing a double check so I will use it to implement a tax report in the loca.